src/Controller/DashboardController.java

```diff
@@ -20,34 +20,67 @@
 
 public class DashboardController {
 	private Simulation sim;
+	private int windowLength = 30;
+	private int ROOM_SIZE = 50;
 
 
 	@FXML private Label currentUser;
+	@FXML private Label currentUserLocation;
+	@FXML private ComboBox<String> currentUserLocationOptions;
```

Comment:
Theres a problem with consistency when theres both a current user label and a combo box because these values can differ.  Maybe we should scrap the label.  Just have the combobox represent the current user's position.  But we'd have to write code to update the combobox when someone edits the user.

Reply:
Yeah the updateDashboard function just needs to update all the values to be blank but I couldn't figure out how to do that (mostly because I kind of ignored it for the time being). Will check how to clear/reset combobox values.